nixos/modules/services/databases/couchdb.nix

```diff
@@ -138,7 +138,7 @@ in {
       };
 
       configFile = mkOption {
-        type = types.string;
+        type = types.str;
```

Comment:
`types.path`

Reply:
I'm not sure about switching to `types.path` for config files, because this has different behavior when the path is unquoted:
```nix
{
  configFile = /var/lib/foo.conf;
}
```

With `types.str` it will be set to `"/var/lib/foo.conf"`, a runtime path, whereas with `types.path` the file will be imported into the store and then that path is used, which can be unsafe if you have passwords stored in the file